packages/cli/src/daemon-config.ts

```diff
@@ -0,0 +1,46 @@
+import { LoggerConfig, SyncOptions } from '@ceramicnetwork/common'
+
+export interface DaemonIpfsConfig {
+  host?: string
+  pinningEndpoints?: string[]
+}
+
+export interface DaemonStateStoreConfig {
+  localDirectory?: string
+  s3Bucket?: string
+}
+
+export interface DaemonHTTPApiConfig {
+  hostname?: string
+  port?: number
+  corsAllowedOrigins?: string | RegExp[]
+}
+
+export interface DaemonCeramicNetworkConfig {
+  name?: string
+  pubsubTopic?: string
+}
+
+export interface DaemonAnchorConfig {
+  anchorServiceUrl?: string
+  ethereumRpcUrl?: string
+}
+
+export interface DaemonCeramicNodeConfig {
+  gateway?: boolean
+  syncOverride?: SyncOptions
+  validateStreams?: boolean
+}
+
+/**
+ * Daemon create options
+ */
+export interface DaemonConfig {
+  anchor?: DaemonAnchorConfig
+  httpApi?: DaemonHTTPApiConfig
```

Comment:
Json normally uses `- ` instead of camelCase, e.g. `http-api`.

Reply:
I don't think that's a valid field name in typescript.  When I try it I get compile failures

Reply:
oh, I guess it works if I put quotes around the field name.  That's pretty annoying though and would need to be done throughout the code anywhere the field is referenced.  Is it worth it?

Reply:
I think it is worth conforming to the widely used json formatting rules.

Reply:
Done, although it really makes things way uglier, and also seems to remove the compile-time checks that the fields being accessed are part of the defined interface, making the code more error-prone.